# Handover Configuration
geometry_msgs/Pose O_T_grasp
geometry_msgs/Pose grasp_T_tool
